### PR TITLE
Refine open API archive docs

### PR DESCRIPTION
**Pull request by Sarangan0219**

### CODE REVIEW

File: en/docs/learn/design-api/create-api/create-a-rest-api-from-an-openapi-definition.md [hunk 1]
```diff
@@ -18,9 +18,27 @@ Follow the instructions below to create a REST API using a Open API definition f
...
+    
+    !!! tip
+        Swagger Definitions should be placed under one root folder and zipped.    
+   
+        **Sample Folder Structure**
+    
+        ```
+            -masterFolder
+              ---Swagger.json
+              ---reference.json
```
Comment (tgtshanika) [inline en/docs/learn/design-api/create-api/create-a-rest-api-from-an-openapi-definition.md:37]: Shall we add a subfolder sample as well? 
ie:
masterFolder
├── reference.yaml
├── schemes
│   └── pet.yaml
└── swagger.yaml

Comment (tgtshanika) [reply en/docs/learn/design-api/create-api/create-a-rest-api-from-an-openapi-definition.md:37]: Shall we attach a sample here?

Comment (Sarangan0219) [reply en/docs/learn/design-api/create-api/create-a-rest-api-from-an-openapi-definition.md:37]: https://github.com/wso2/docs-apim/pull/1728/commits/99332a4afda62cd5af0e1b492520e36386c96eed

**Review by Mariangela — APPROVED** [review]

File: en/docs/learn/design-api/create-api/create-a-rest-api-from-an-openapi-definition.md (hunk 1 above)
Comment (praminda) [reply en/docs/learn/design-api/create-api/create-a-rest-api-from-an-openapi-definition.md:37]: is `masterFloder` a required filed name for something or just a sample name? if its a sample shall we rename it to something like `rootDir`

File: en/docs/learn/design-api/create-api/create-a-rest-api-from-an-openapi-definition.md [hunk 2]
```diff
@@ -18,9 +18,34 @@ Follow the instructions below to create a REST API using a Open API definition f
...
+        **Sample Folder Structures**
+    
+        ```
+            -masterFolder
+              ---Swagger.json
+              ---reference.json
+              ---add.json
+        ```
+        ```
+            -masterFolder
```
Comment (praminda) [inline en/docs/learn/design-api/create-api/create-a-rest-api-from-an-openapi-definition.md:41]: same here

File: en/docs/learn/design-api/create-api/create-a-rest-api-from-an-openapi-definition.md (hunk 1 above)
Comment (Sarangan0219) [reply en/docs/learn/design-api/create-api/create-a-rest-api-from-an-openapi-definition.md:37]: It is not mandatory to have a specified name for masterFolder. We did not mention that as a convention to have a name like **masterFolder** . So IMO we don't need to rephrase it.

WDYT ?

File: en/docs/learn/design-api/create-api/create-a-rest-api-from-an-openapi-definition.md (hunk 2 above)
Comment (Sarangan0219) [reply en/docs/learn/design-api/create-api/create-a-rest-api-from-an-openapi-definition.md:41]: It is not mandatory to have a specified name for masterFolder. We did not mention that as a convention to have a name like masterFolder . So IMO we don't need to rephrase it.

WDYT ?

**Pull request merged**